cardinal/ecs/store/store.go

```diff
@@ -17,6 +17,8 @@ import (
 	"pkg.world.dev/world-engine/cardinal/ecs/storage"
 )
 
+var _ IManager = &Manager{}
```

Comment:
should the struct be public? it seems like we can just have

```go
type Manager interface { 
...
}

type manager struct {
...
}
```

Reply:
next PR is fine, just want to make sure we dont have both a struct and interface exposed